ws-nextjs-app/pages/ws/languages.page.tsx

```diff
@@ -10,6 +15,62 @@ export const getServerSideProps: GetServerSideProps = async context => {
     'public, stale-if-error=300, stale-while-revalidate=120, max-age=30',
   );
 
+  const isTestEnvironment = context.query.renderer_env === 'test';
+  const pageType = isTestEnvironment ? HOME_PAGE : STATIC_PAGE;
+  const service = isTestEnvironment ? 'pidgin' : 'ws';
+
+  if (isTestEnvironment) {
+    const initialData = await getInitialData({
```

Comment:
Let's simplify this: we have a function to check if we're on a live environment: https://github.com/bbc/simorgh/blob/latest/src/app/lib/utilities/isLive/index.js

It means we won't see the static page on local or test, but we will see it on live. This is fine whilst we're working on the global homepage.

 ```suggestion
  let pageType = STATIC_PAGE;
  let service = 'ws';

  if (!isLive()) {
    pageType = HOME_PAGE;
    service = 'pidgin';
 
    const initialData = await getInitialData({
```